data-processing/entities/definitions/commands/detect_definitions.py

```diff
@@ -619,19 +639,33 @@ def process(
 
                     # Detect terms and definitions in each sentence with a pre-trained definition
                     # extraction model, from the featurized text.
-                    intents, slots, slots_confidence = model.predict_batch(
-                        cast(List[Dict[Any, Any]], features)
+                    termdef_intents, termdef_slots, termdef_slots_confidence = model.predict_batch(
```

Comment:
Is there ever a circumstance where we will not want to just get everything from `model.predict_batch` for each of the three tasks at the same time? Could we just make model.predict_batch return predictions for all three tasks at once? (Mostly curious. I realize this could potentially require a big change, which might not be worth it.)

Reply:
I think the expectation is for the 3 model paradigm to go away quickly. So, my changes mostly are to make sure that change will be very very easy (just deleting 2 of the 3 models)